x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/connectors/slack.ts

```diff
@@ -211,7 +211,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         //     tags: '',
         //   },
         // ]);
-        // const toastTitle = await toasts.closeToast();
+        // const toastTitle = await toasts.dismiss();
```

Comment:
It's just a commented out section, so it's not critical here. However, I think the correct replacement for the method would be `getTitleAndDismiss`.